### PR TITLE
[17.0][MIG] account_financial_risk: Migration to 17.0

### PR DESCRIPTION
**Pull request by Mat-moran**
- Change version in manifest
- Adapt views to version 17.0

### CODE REVIEW

**Comment by Mat-moran** [comment]
Hi, @pedrobaeza can you help on this? we have a  problem with the field `amount_residual_currency` when the method`_get_amount_in_risk_currency()` is executed.

The field `amount_residual_currency` when is defined it explicetily as
```python
amount_residual_currency = fields.Monetary(
    string='Residual Amount in Currency',
    compute='_compute_amount_residual', store=True,
    group_operator=None, # This is making the function fail
    help="The residual amount on a journal item expressed in its currency (possibly not the "
         "company currency).",
)
```

what do you sugest?

**Comment by pedrobaeza** [comment]
Sorry, I can't say, as I haven't dug on it.

File: account_financial_risk/data/account_financial_risk_data.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" ?>
+<odoo noupdate="1">
+        <!-- Account Tax Group -->
+        <record id="tax_group_taxes" model="account.tax.group">
```
Comment (carlosdauden) [inline account_financial_risk/data/account_financial_risk_data.xml:4]: Why this module adds new tax group?

File: account_financial_risk/models/res_partner.py [hunk 2]
```diff
@@ -354,7 +354,6 @@ def _compute_risk_account_amount(self):
                 group["domain"] + [("partner_id", "in", customers.ids)],
                 group["fields"],
                 group["group_by"],
-                orderby="id",
```
Comment (carlosdauden) [inline account_financial_risk/models/res_partner.py]: Use _read_group to remove this

File: account_financial_risk/data/account_financial_risk_data.xml (hunk 1 above)
Comment (carlosdauden) [inline account_financial_risk/data/account_financial_risk_data.xml:4]: Why this module adds new tax group?

**Review by carlosdauden — CHANGES_REQUESTED** [review]
Ignore not related files as macOS.pkg

Take a look at:
https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#7git

File: account_financial_risk/data/account_financial_risk_data.xml (hunk 1 above)
Comment (Mat-moran) [reply account_financial_risk/data/account_financial_risk_data.xml:4]: Thanks for the review,

This was in odoo account module but has been removed in v17. As it is required by the test, so we added to this module.

```xml
# before v17 this data was on -> addons/account/data/account_data.xml
<!-- Account Tax Group -->
<record id="tax_group_taxes" model="account.tax.group">
    <field name="name">Taxes</field>
    <field name="sequence">0</field>
</record>
```

File: account_financial_risk/models/res_partner.py (hunk 2 above)
Comment (Mat-moran) [reply account_financial_risk/models/res_partner.py]: I am not sure if I am understanding. 

the return of `_read_group` is not the same as `read_group`

**Comment by ioans73** [comment]
@Mat-moran what is the status of this?

File: account_financial_risk/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# Copyright 2016-2018 Tecnativa - Carlos Dauden
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Account Financial Risk",
+    "summary": "Manage customer risk",
+    "version": "17.0.1.0",
```
Comment (sbiosca-s73) [inline account_financial_risk/__manifest__.py:7]: Please, put the version in the correctly form 
`17.0.1.0.0`

Comment (pedrobaeza) [inline account_financial_risk/__manifest__.py:7]:
```suggestion
    "version": "17.0.1.0.0",
```

File: account_financial_risk/data/account_financial_risk_data.xml (hunk 1 above)
Comment (pedrobaeza) [reply account_financial_risk/data/account_financial_risk_data.xml:4]: You should fix the test, not to add useless data.

File: account_financial_risk/models/res_partner.py (hunk 2 above)
Comment (carlosdauden) [reply account_financial_risk/models/res_partner.py]: Look this:
https://github.com/OCA/credit-control/pull/377#pullrequestreview-2035518788

**Comment by pedrobaeza** [comment]
Superseded by #379

**Pull request closed without merging**